nsaway/utils.py

```diff
@@ -46,29 +50,24 @@ def local_import(name, globals=None, locals=None, fromlist=None):
         if fp:
             fp.close()
 
-class LogLevel:
-  INFO = "I" # Just some info, almost useless
-  WARN = "W" # Warning situation on the system
-  ERROR = "E" # Errors running nsaway
-  NOTICE = "N" # Notice about some unwanted action on nsaway
-
-"""
-Write message in the log file
-"""
-def log(level, msg):
-  contents = '[{0}] {1} | {2}\n'.format(str(datetime.now()), level, msg)
-  with open(LOG_FILE, 'a+') as log:
-    log.write(contents)
+def create_timed_rotating_log(path):
+    logger.setLevel(logging.NOTSET)               # See https://docs.python.org/2/library/logging.html#levels
+    handler = TimedRotatingFileHandler(LOG_FILE,  # https://docs.python.org/2/library/logging.handlers.html#timedrotatingfilehandler
+                                       when="midnight",
+                                       interval=1,
+                                       backupCount=30)
+    handler.suffix = "%Y%m%d"
+    logger.addHandler(handler)
 
 """
 Write message in the log file and exit with message
 """
 def exit_log(level, msg):
```

Comment:
Must remove the level arg (from nsaway.py function exit_log is called with 1 parameter)
Can also delete line 66 and change line 68 to the following:

`sys.exit("[ERROR] {0}".format(msg))`